subsys/bluetooth/mesh/cfg.c

```diff
@@ -79,6 +82,15 @@ static enum bt_mesh_feat_state feature_get(int feature_flag)
 		       BT_MESH_FEATURE_DISABLED;
 }
 
+static int check_proxy_adv(struct bt_mesh_subnet *sub, void *cb_data)
+{
+	if (sub->node_id == BT_MESH_NODE_IDENTITY_RUNNING) {
+		return 1;
+	}
+
+	return 0;
+}
```

Comment:
Very minor: perhaps this would be even easier to follow here and below if this function is just called `node_id_is_running`?

That way, line 105 reads like plain English: `bt_mesh_subnet_find(node_id_is_running, NULL)`, and it's easy to verify that this function does what it's meant to.

The content could also be reduced to `return (sub->node_id == BT_MESH_NODE_IDENTITY_RUNNING);`